examples/files/files.pony

```diff
@@ -5,8 +5,9 @@ actor Main
     let caps = recover val FileCaps.>set(FileRead).>set(FileStat) end
 
     try
+      let auth = env.root as AmbientAuth
       with file = OpenFile(
-        FilePath(env.root as AmbientAuth, env.args(1)?, caps)?) as File
+        FilePath(auth, env.args(1)?, caps)) as File
```

Comment:
I think this change with let auth temporary variable should be reverted. it hides the "actual change".